### PR TITLE
feature/SOF-4711 - Energy above hull of bulks

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+                'is_hubbard (unitless)',
+                'energy_per_atom (eV)',
+                'exfoliation_energy_per_atom (eV/atom)',
+                'is_bandgap_direct (unitless)',
+                'is_metal (unitless)',
+                'energy_vdw_per_atom (eV/atom)',
+                'total_magnetization (Bohr Magneton)']
+target_column = ['bandgap (eV)']
+matpedia_id = ['2dm_id (unitless)']
+atoms_col = ['atoms_object (unitless)']
```
Comment (timurbazhirov) [inline DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py]: Same things as before:

- shortnames should be avoided: `decomp` -> `decomposition` etc., `col` -> `column`
- no explanations in this notebook

Comment (AcylSilane) [reply DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py]: Renamed the requested variables, and added some comments

Comment (AcylSilane) [reply DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py]: Made those changes, and added notes

File: DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+import matplotlib.pyplot as plt
+import sklearn.impute
+import seaborn as sns
+
+import sys
+sys.path.append("../../../")
+import DigitalEcosystem.utils.figures
+import DigitalEcosystem.utils.misc
+import DigitalEcosystem.utils.functional
+from DigitalEcosystem.utils.misc import noble_gases, fblock, d_synths, matminer_descriptors
```
Comment (timurbazhirov) [inline DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py]: `d_synth` -> `synthetic_elements_in_d_block`

File: DigitalEcosystem/utils/misc.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    'average_cn',
+    'global_instability',
+    'perimeter_area_ratio',
+    'ewald_energy_per_atom',
+    'structural complexity per atom',
+    'structural complexity per cell',
+    'n_symmetry_ops'
+
+]
+noble_gases = ['He', 'Ne', 'Ar', 'Kr', 'Xe', 'Rn']
```
Comment (timurbazhirov) [inline DigitalEcosystem/utils/misc.py]: Maybe `element_symbols.f_block`, `element_symbols.d_synthetics`, element_symbols.noble_gases`

Comment (AcylSilane) [reply DigitalEcosystem/utils/misc.py]: Done

File: DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py (hunk 2 above)
Comment (AcylSilane) [reply DigitalEcosystem/exploratory/matproj_parent_eAboveHull/e_above_hull.py]: Changed

**Comment by AcylSilane** [comment]
Merging per https://exabyte.atlassian.net/browse/SOF-4711?focusedCommentId=42198

**Pull request merged**